horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/RunService.java

```diff
@@ -426,10 +426,10 @@ class RunSummary {
       public int id;
       @JsonProperty(required = true)
       @Schema(description = "Start timestamp", example = "1698013206000")
-      public long start;
+      public Instant start;
       @JsonProperty(required = true)
       @Schema(description = "Stop timestamp", example = "1698013206000")
-      public long stop;
+      public Instant stop;
```

Comment:
Horreum currently supports passing timestamps in Runs as `Long`, numeric value as `String`, and Strings formatted as `ISO_DATE_TIME` 

 https://github.com/Hyperfoil/Horreum/blob/f66f0c9eb69f6b1e4e927cd4871e685fdcd3cff8/horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/RunServiceImpl.java#L516
 
The Api treats timestamps and Long. We need to decide on a way of representing the timestamps in the API that presents a consistent User experience.  

I think parsing the timestamps for Runs is fine and should continue to be supported, but all API input output should be an instant, but formatted to a String with `ISO_DATE_TIME` format. 

The examples should reflect this